dll/cpl/access/lang/cs-CZ.rc

```diff
@@ -109,155 +117,155 @@ END
 
 IDD_STICKYKEYSOPTIONS DIALOGEX 0, 0, 246, 220
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Sticky Keys Settings"
+CAPTION "Nastavení funkce Jedním prstem"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Keys", -1, 6, 11, 234, 62
-    LTEXT "Press the SHIFT-Key 5 times in order to activate the Sticky Keys feature.",
+    GROUPBOX "Klávesy", -1, 6, 11, 234, 62
+    LTEXT "Aktivujte funkci Jedním prstem stisknutím klávesy SHIFT 5krát.",
```

Comment:
The last word(s) seems to be ill-formatted.

Reply:
"5krát" is normally formed word. (https://prirucka.ujc.cas.cz/?id=790)

Reply:
thanks; that's an interesting rule as usually other languages put some sort of "separator" between the numeral and the text (unit....) that follows.